src/visualization/types/Table/TableCell.tsx

```diff
@@ -272,12 +273,9 @@ class TableCell extends PureComponent<Props> {
       return (this.fieldName || '').toString()
     }
 
-    if (
-      !isNaN(+data) &&
-      decimalPlaces.isEnforced &&
-      decimalPlaces.digits < 100
-    ) {
-      return (+data).toFixed(decimalPlaces.digits)
+    if (!isNaN(+data)) {
```

Comment:
This is clearly an edge case. Be careful with `isNaN` which is short for `window.isNaN`. Especially in combination with the casting operator here. You are coercing non-numbers into numbers and then checking if they are `NaN` or not. An empty string gets coerced into `0` which is a number. Are you sure that empty strings should be treated as `0` rather than just simply keeping them as an empty string?

After you remove the casting operator, please also look into the `isNaN` usage, because `isNaN` will do its own coercing and you will end up again with the case of empty string being forced into being `0`.

There's double coercion going on here. If empty string should not be treated as `0` please remove both usages of the coercion.

Reply:
this is out of scope for this fix.  after the giraffe pr gets merged in (a duplicate of this one, essentially), will apply those same changes here to fix empty strings from incorrect coercion.